exercises/concept/squeaky-clean/src/test/java/SqueakyCleanTest.java

```diff
@@ -43,50 +43,50 @@ public void leading_and_trailing_spaces() {
 
     @Test
     @Tag("task:2")
-    @DisplayName("The clean method replaces control characters with CTRL")
-    public void ctrl() {
-        assertThat(SqueakyClean.clean("my\0\r\u007FId")).isEqualTo("myCTRLCTRLCTRLId");
+    @DisplayName("The clean method converts kebab to camel case after removing a dash")
+    public void kebab_to_camel_case() {
+        assertThat(SqueakyClean.clean("à-ḃç")).isEqualTo("àḂç");
```

Comment:
This test should probably use `a-bc` and `aBc` like in the exercise instructions.

Reply:
Done!